scripts.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var questions = {
+        strong: 'Do ye like yer drinks strong?',
+        salty: 'Do ye like it with a salty tang?',
+        bitter: 'Are ye a lubber who likes it bitter?',
+        sweet: 'Would ye like a bit of sweetness with yer poison?',
+        fruity: 'Are ye one for a fruity finish?',
+    },
+    ingredients = {
+        strong: [
+            'Glug of rum',
+            'Slug of whisky',
+            'Splash of gin'
+        ],
+        salty: [
+            'Olive on a stick',
+            'Salt-dusted rim',
+            'Rasher of bacon'
+        ],
+        bitter: [
+            'Shake of bitters',
+            'Splash of tonic',
+            'Twist of lemon peel'
+        ],
+        sweet: [
+            'Sugar cube',
+            'Spoonful of honey',
+            'Splash of cola'
+        ],
+        fruity: [
+            'Slice of orange',
+            'Dash of cassis',
+            'Cherry on top'
+        ],
+    },
+    preferences = {};
+
+var Bartender = function() {
+    this.createDrink = function() {
```

Comment:
Think you might be trying to use `this.createDrink` as a prototype but that won't work here. You are declaring that `this.createDrink is an anonymous function`. You won't be able to apply `this.createDrink` to your `Bartender` object. 
You would want to day:
```
var Bartender = function(x, y, z) {
 this.thing1 = x,
 this.thing2 = y,
 this.thing3 = z
}

Bartender.prototype.createDrink = function(params) {
 // code here for creating your drink
}

```